iot_verifier/src/loader.rs

```diff
@@ -201,14 +201,16 @@ impl Loader {
         // widen the window for these over that used for the beacons
         // this is to allow for a witness being in a rolled up file
         // from just before or after the beacon files
+        // the width extention needs to be at least equal to that
+        // of the ingestor roll up time
         // for witnesses we do need the filter but not the arc
         match self
             .process_events(
                 FileType::IotWitnessIngestReport,
                 &self.ingest_store,
                 gateway_cache,
-                after - ChronoDuration::seconds(60 * 2),
-                before + ChronoDuration::seconds(60 * 2),
+                after - ChronoDuration::seconds(60 * 6),
```

Comment:
Should this be 10 minutes to follow the 2x roll up guideline?

also why not use the minute constructor on ChronoDuration?

Reply:
i suspect one 1x is sufficient.  Each multiple involves doubling the amount of files we have to process, so we should only extend if necessary

re chrono seconds vs minutes...no reason...just what has been there.  I can change to minutes to give you a good friday